test/micro-prometheus.ts

```diff
@@ -0,0 +1,42 @@
+import test from 'ava'
+import { Server } from 'http'
+import { join } from 'path'
+import * as rp from 'request-promise-native'
+import * as stdMock from 'std-mocks'
+import n9Micro from '../src'
+
+const closeServer = async (server: Server) => {
+	return new Promise((resolve) => {
+		server.close(resolve)
+	})
+}
+
+const MICRO_FOO = join(__dirname, 'fixtures/micro-prometheus/')
+
+test('Basic usage, create http server', async (t) => {
+	stdMock.use()
+	const { app, server } = await n9Micro({
+		path: MICRO_FOO,
+		http: { port: 5000 },
+		prometheus: { port: 5001, accuracies: ['s'] }
+	})
+	let res = await rp({ uri: 'http://localhost:5000/sample-route', resolveWithFullResponse: true, json: true })
+	t.is(res.statusCode, 204)
+	t.is(res.body, undefined)
+
+	res = await rp({ uri: 'http://localhost:5000/by-code/code1', resolveWithFullResponse: true, json: true })
+	t.is(res.statusCode, 204)
+	t.is(res.body, undefined)
+
+	// Check /foo route added on foo/foo.init.ts
+	const resProm = await rp({ uri: 'http://localhost:5001/' })
+	t.true(resProm.includes('http_requests_total{method="get",path="/sample-route",status_code="204"} 1'), 'Prom exposition contains call to sample-route')
+	t.true(resProm.includes('http_requests_total{method="get",path="/by-code/code1",status_code="204"} 1'), 'Prom exposition contains call with route pattern')
```

Comment:
`/by-code/code1` is not the expected metric.
The expected metric is `/by-code/:code`, this is to group similar calls (by default at least).

Reply:
Ok understood as we discussed I added the normalizePath and prepended the test path by /:version(v\\\\d+)?/